src/main/java/mServer/crawler/sender/wdr/WdrUrlLoader.java

```diff
@@ -0,0 +1,37 @@
+package mServer.crawler.sender.wdr;
+
+import de.mediathekview.mlib.tool.MVHttpClient;
+import java.io.IOException;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class WdrUrlLoader {
+    
+    private final Logger LOG = LogManager.getLogger(WdrUrlLoader.class);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "LOG" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)